packages/run-protocol/src/makeTracer.js

```diff
@@ -1,13 +1,28 @@
 let debugInstance = 1;
 
-const makeTracer = name => {
+const makeTracer = (name, enable = true) => {
   debugInstance += 1;
   let debugCount = 1;
   const key = `----- ${name}.${debugInstance} `;
-  const debugTick = (...args) => {
-    console.log(key, (debugCount += 1), ...args);
-  };
-  return debugTick;
+  // the cases below define a named variable to provide better debug info
+  switch (enable) {
+    case false: {
+      const logDisabled = (..._args) => {};
+      return logDisabled;
+    }
+    case 'info': {
+      const infoTick = (...args) => {
+        console.info(key, (debugCount += 1), ...args);
+      };
+      return infoTick;
+    }
+    default: {
```

Comment:
`info` uses `console.info`; `default` uses `console.log`. So Info I believe is normally not on.

Reply:
> So Info I believe is normally not on

It's often disabled but Chrome, for example, defaults `info` to on:
<img width="179" alt="Screen Shot 2022-03-24 at 4 15 49 PM" src="https://user-images.githubusercontent.com/21505/160025161-7c102ee2-db85-49eb-aef8-3c9c83e9ba49.png">

> info uses console.info; default uses console.log

In Node, console.info is an alias for console.log.  ([ref](https://nodejs.org/api/console.html#consoleinfodata-args))

Maybe this will all get sorted out in the emerging [Console Standard](https://console.spec.whatwg.org/#loglevel-severity).